extensions/amp-bodymovin-animation/amp-bodymovin-animation.md

```diff
@@ -61,14 +61,11 @@ for an action to start playing.
 
 ### `renderer` (optional)
 
-By default, this component uses the SVG renderer, this uses a light version of
-the Bodymovin animation player. However, if developers feel that they need the
-full player and want to use an HTML renderer they may do so by specifying the
-`renderer` attribute to be `html`.
+Bodymovin animation player supports multiple renderers. By default, the SVG
+renderer is used. Renderer can be changed to HTML or Canvas by setting this
+property to `html` and `canvas` respectively.
```

Comment:
One final nit, I think it's okay to let the external documentation explain what this means. 

```suggestion
Multiple [renderers](https://airbnb.io/lottie/#/web?id=usage) are supported and can be specified:

-   `canvas`
-   `html`
-   `svg` (default)
```